### PR TITLE
Chore - Remove activity content

### LINKED ISSUE #186: Use Activity Content SDK in SDK-ts
- [ ] Remove any sdk-ts functions that are just duplicates of Activity Content functions
- [ ] Replace any internal function calls with Activity Content SDK calls

### PR DESCRIPTION
**Pull request by dopatraman**
Problem
=======
The purpose of this PR is to replace the activity content code in the SDK repo with the activity content NPM package.

Closes #186

### CODE REVIEW

**Review by shannonwells — APPROVED** [review]
Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - BREAKING: Changed createRegistration to now longer return a user URI and instead returns a Registration Object.
 - Changed `convertToDSNPUserId` function to throw correct errors
 - Updated building to Node 16
+- Replaced activity content folder with standalone activity content NPM module
```
Comment (wilwade) [inline CHANGELOG.md:11]: @dopatraman this should likely be marked as breaking with notes on how to update as we also (correctly) removed the entire `activityContent` export as well as `generators/activityContent`.

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "@dsnp/test-generators": "^0.1.0",
+    "@dsnp/activity-content": "^1.0.0",
```
Comment (wilwade) [inline package.json:36]: Try v1.0.1

**Review by wilwade — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dopatraman) [reply CHANGELOG.md:11]: Will update. Just a heads up though, `generators/activityContent` has not been removed -- it just wraps the NPM package.

**Review by wilwade — APPROVED** [review]

**Pull request merged**